caffe2/operators/mean_op.cu

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2016-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "caffe2/core/context_gpu.h"
+#include "caffe2/operators/mean_op.h"
+
+namespace caffe2 {
+
+REGISTER_CUDA_OPERATOR(Mean, MeanOp<CUDAContext>);
+
+template <>
+bool MeanOp<CUDAContext>::RunOnDevice() {
```

Comment:
Should this function be in the .h file directly? It seems to be identical with the cpu version.